configs/components/facter.rb

```diff
@@ -109,10 +109,13 @@
       skip_blkid = 'OFF'
     end
   end
+  if platform.is_huaweios?
+    skip_blkid = 'ON'
```

Comment:
@melissa  on line 100, the platform.is_deb? would be true for huaweios and that code branch would be activated without checking any of the later tests. So that wouldn't work.